README.md

```diff
@@ -57,5 +57,28 @@ From an LmP device:
 ~~~
 
 # Testing/Troubleshooting
-This project includes a simple `fake-lmp-device-register` that can be used
-to quickly test this project's API.
+
+For scenarios where you'd like to test the API call without involving a physical device or emulation:
+
+This project includes a simple [fake-lmp-device-register](./fake-lmp-device-register) within this project. 
+This tool mimics the behavior of `lmp-device-register`, enabling you to quickly test this project's API.
+
+## Running the fake-lmp-device-register
+
+### Dependencies:
+- Python 3
+- requests library
+- openssl command-line tool
+
+### Run:
+
+You can install the requests library using pip:
+```shell
+pip3 install requests
+```
+
+```shell
+python3 fake-lmp-device-register --registration-url "http://<IP of docker-compose host>:80/sign" --factory <factory>
+``` 
```

Comment:
TODO: Check why it is not working. 

```
$ python3 fake-lmp-device-register --registration-url "http://localhost:80/sign" --factory camila-test-arm64
 private key from -: No such file or directory
Traceback (most recent call last):
  File "/Users/camilamacedo/go/src/github.com/foundriesio/factory-registration-ref/fake-lmp-device-register", line 146, in <module>
    main(options)
  File "/Users/camilamacedo/go/src/github.com/foundriesio/factory-registration-ref/fake-lmp-device-register", line 65, in main
    pkey, csr = create_key(opts.uuid, opts.factory, opts.production)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/camilamacedo/go/src/github.com/foundriesio/factory-registration-ref/fake-lmp-device-register", line 59, in create_key
    r.check_returncode()
  File "/opt/homebrew/Cellar/python@3.11/3.11.6/Frameworks/Python.framework/Versions/3.11/lib/python3.11/subprocess.py", line 502, in check_returncode
    raise CalledProcessError(self.returncode, self.args, self.stdout,
subprocess.CalledProcessError: Command '['openssl', 'req', '-new', '-config', '/var/folders/zr/51746skj0gz6qnt27hmvvbnc0000gn/T/tmp99312fzn', '-key', '-']' returned non-zero exit status 1.
````

Reply:
the problem is `-key -` because it is literally the value `-`. Not really meaning `stdin`. As far as I can tell from the docs. So try to write the key to another temp file and then reference that.

Reply:
But that should work, right?
We have a bug I just tracked here for do not forget 
I will look at that after.

Reply:
that command should work,I was referencing the code of `fake-lmp-device-register` itself. That was unclear of me, I apologize.

Reply:
I understand that the fake-lmp-device-register is to help out we debug this API
So, I was trying to use. No problem at all. 
I had not made the effort to check it further yet. Thank you a lot for the help.

Reply:
As we are promoting an exact command to run in the README, we need to make sure that it works.

Also, a good addition would be to tell that the user can verify the connectivity of the fake device by running `$SOTA_DIR/curl` after the registration.

Reply:
@vkhoroz the fix for this script is in the PR: https://github.com/foundriesio/factory-registration-ref/pull/16